deps.edn

```diff
@@ -310,7 +310,7 @@
     ;; It fixes an issue where TypeScript can't parse the `cljs_env.js` output file.
     org.clojure/google-closure-library {:mvn/version "0.0-20230227-c7c0a541"}
     refactor-nrepl/refactor-nrepl      {:mvn/version "3.6.0"}
-    thheller/shadow-cljs               {:mvn/version "2.22.2"}}}
+    thheller/shadow-cljs               {:mvn/version "2.24.0"}}}
```

Comment:
This is not related to the change here, but I was running into some issues with the Cljs REPL so I tried bumping to the latest version of shadow-cljs to see if it fixed things, ultimately didn't change things but it seems like updating shadow is something that's good to do anyway so I'm leaving it in.